<launch>
  <!-- laser.launch, created by: William Willie Wells, in May 2016 -->

  <!--================================VICON Object====================================-->
  <!--group ns="a">
    <node pkg="vicon_pose" type="vicon_pose" name="conv_q_p" output="screen">
      <param name="object_name" value="JENNY" type="str" />
    </node>
  </group-->

  <!--======================MIT AscTec Protocol and Communication=====================-->
    <!-- movement at constant velocity -->
  <include file="$(find gps_testing)/launch/hummingbird_indoor.launch" />
  <!--include file="$(find gps_testing)/launch/hummingbird_outdoor.launch" /-->
  <!--include file="$(find gps_testing)/launch/firefly_outdoor.launch" /-->

  <!--================================Bag Recording===================================-->  
  <group ns="a">
  <!--node name="flight_logger" pkg="flight_logger" type="flight_logger.py" output="screen" />
  <node name="flight_log_processor" pkg="flight_logger" type="flight_log_processor.py" output="screen" /-->
  <include file="$(find collab_launch)/common/record.launch" />
  <!-- sed -i.bak s/unl_comp_sci#/crash#+1/ path/bat -->
  <!-- ./path/bat -->

  <!--==============================Arduino Communication============================-->
    <node name="arduino_comm" type="serial_comm" pkg="communication">
      <remap from="serial_rx_data" to="device_rx_data"/>
      <remap from="serial_tx_data" to="device_tx_data"/>
      <param name="serial_baudrate" value="9600"/>
      <param name="serial_device" value="/dev/ttyACM0"/> 
    </node>

    <node name="parse_serial" type="parse_serial" pkg="gps_testing" output="screen"></node>

  <!--====================================Main Node==================================-->

    <!--node name="nimbus_gui" type="nimbus_gui.py" pkg="nimbus_gui" output="screen">
      <param name="min_y" value="-2.0"/>
      <param name="max_y" value="2.0"/>
      <param name="min_x" value="-2.0"/>
      <param name="max_x" value="2.0"/>
      <param name="min_z" value="0.0"/>
      <param name="max_z" value="2.5"/>
    </node--> 
  </group>
</launch>
